#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: in Debian we're including a couple of other files in
## DP: config.inc.php, and their changes will never be picked up
## DP: if we leave this check in.

@DPATCH@

diff -ur phpMyAdmin-3.1.0-all-languages.orig/libraries/Config.class.php phpMyAdmin-3.1.0-all-languages/libraries/Config.class.php
--- phpMyAdmin-3.1.0-all-languages.orig/libraries/Config.class.php	2008-11-28 14:21:00.000000000 +0100
+++ phpMyAdmin-3.1.0-all-languages/libraries/Config.class.php	2008-11-30 11:49:37.000000000 +0100
@@ -298,15 +298,20 @@
      */
     function __wakeup()
     {
+        /* Disable in Debian: we're including a couple of other files
+         * in config.inc.php, and their changes will never be picked up
+         * if we leave this check in.
         if (! $this->checkConfigSource()
           || $this->source_mtime !== filemtime($this->getSource())
           || $this->default_source_mtime !== filemtime($this->default_source)
           || $this->error_config_file
           || $this->error_config_default_file) {
+         */
             $this->settings = array();
             $this->load();
             $this->checkSystem();
-        }
+         /*
+        } */
 
         // check for https needs to be done everytime,
         // as https and http uses same session so this info can not be stored
